hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationServiceImpl.java

```diff
@@ -135,52 +135,51 @@
     // contains the current executing asyncOperations. This information is needed for the operation-ping.
     // this is a temporary solution till we found a better async operation abstraction
     @Probe
-    private final Set<Operation> asyncOperations
-            = newSetFromMap(new ConcurrentHashMap<Operation, Boolean>());
+    private final Set<Operation> asyncOperations = newSetFromMap(new ConcurrentHashMap<Operation, Boolean>());
 
+    @SuppressWarnings("checkstyle:executablestatementcount")
```

Comment:
The `HazelcastProperties properties = node.getProperties();` is the new statement, which pushed us over the limit.